backend/private-graph/graph/resolver.go

```diff
@@ -487,6 +493,10 @@ func (r *Resolver) isAdminInProject(ctx context.Context, project_id int) (*model
 	if err != nil {
 		return nil, e.Wrap(err, "error querying projects")
 	}
+	if len(projects) == 0 {
```

Comment:
If you have no projects then you're not logged in. We short circuit here because if we continued, we would return a `AuthorizationError` which wouldn't be semantically the same.